src/mqt/predictor/rl/helper.py

```diff
@@ -544,22 +515,19 @@ def get_bqskit_native_gates(device: Device) -> list[gates.Gate] | None:
     Returns:
         The native gates of the given provider.
     """
-    provider = device.name.split("_")[0]
-
     native_gatesets = {
-        "ibm": [gates.RZGate(), gates.SXGate(), gates.XGate(), gates.CNOTGate()],
-        "rigetti": [gates.RXGate(), gates.RZGate(), gates.CZGate()],
-        "ionq": [gates.RXXGate(), gates.RZGate(), gates.RYGate(), gates.RXGate()],
+        "ibm_heron": [gates.RZGate(), gates.SXGate(), gates.XGate(), gates.CZGate()],
+        "ibm_eagle": [gates.RZGate(), gates.SXGate(), gates.XGate(), gates.ECRGate()],
+        "ibm_falcon": [gates.RZGate(), gates.SXGate(), gates.XGate(), gates.CXGate()],
         "quantinuum": [gates.RZZGate(), gates.RZGate(), gates.RYGate(), gates.RXGate()],
         "iqm": [gates.U3Gate(), gates.CZGate()],
```

Comment:
Shouldn't these be somehow inherited from MQT Bench?
At least they do not currently match the MQT Bench gate sets for the respective devices perfectly (e.g. IQM)

MQT Bench provides the names of the gates and one could have a dictionary of known gates and their BQSKit equivalent.

Reply:
Nice 👍🏻 
I suppose BQSKit does not support the R gate? Because I don't see it in the gate map.
Is this all the gates that BQSKit supports?